### PR TITLE
Setup semantic release

### PR DESCRIPTION
**Pull request by RockslideJr**
[API-4326](https://vajira.max.gov/browse/API-4326)

This adds [semantic-release](https://github.com/semantic-release/semantic-release) as part of our CI/CD pipeline. With this in place, any build on master will check to see if a new version needs to be released. If so, it will build and release it as well as push the git tags and release back to the repo.

semantic-release uses [formatted commit messages](https://github.com/semantic-release/semantic-release#commit-message-format) to determine when a new release is needed. We have a [follow-up ticket](https://vajira.max.gov/browse/API-4912) to determine and enforce how we want to make these formatted message into our workflow, so for now we won't have a release version.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "loast",
   "description": "CLI for testing Lighthouse APIs using OpenAPI specs",
-  "version": "0.2.0",
+  "version": "0.0.0-development",
```
Comment (alexandervalencia) [inline package.json:4]: Where did the decision come for this to be the initial version? Is it a "semantic release" standard?

**Review by alexandervalencia — COMMENTED** [review]
LGTM 🚀

File: package.json (hunk 1 above)
Comment (RockslideJr) [reply package.json:4]: Yes, this is what semantic release sets your version to when you initialize it.

**Review by alexandervalencia — APPROVED** [review]
This is approved 👨‍⚖️

**Pull request merged**